docs/ref/contrib/postgres/fields.txt

```diff
@@ -586,6 +586,39 @@ containment and keys with :class:`~django.contrib.postgres.fields.HStoreField`.
 
 .. _range-fields:
 
+``CITextField``
+===============
+
+.. class:: CITextField(**options)
+
+    .. versionadded:: 1.11
+
+    This field is in essence a :class:`~django.db.models.CharField` with a few
+    important details:
+
+    1. You don't need to specify lower when querying against data to force use
+       of an index.
+
+    2. An index is used if you specify `UNIQUE` or `PRIMARY KEY`, unlike
+       :class:`~django.db.models.CharField`.
+
+    3. There's no need to create a functional index to enforce use of `lower`
+       in the queries.
+
+    To use this field,
+
+    1. Setup the citext extension in PostgreSQL before the first ``CreateModel``
+       :class:`~django.contrib.postgres.operations.CITextExtension` operation.
+       The code to setup the extension is similar to the example for configuring
+       the :class:`~django.contrib.postgres.fields.HStoreField`; the instructions
+       to setup `HStoreField` also reflect the requirements to get support for
+       `CITextField`
+
+    There are possible performance considerations to be made prior to using this
```

Comment:
"Read about the performance considerations prior to using this field."
link "performance considerations" and chop the second sentence.